apis/management.cattle.io/v3/cluster_scan_types.go

```diff
@@ -78,3 +78,29 @@ type ClusterScan struct {
 	Spec   ClusterScanSpec   `json:"spec"`
 	Status ClusterScanStatus `yaml:"status" json:"status,omitempty"`
 }
+
+type CisBenchmarkVersionInfo struct {
+	MinKubernetesVersion string `yaml:"min_kubernetes_version" json:"minKubernetesVersion"`
```

Comment:
Do you envision use of MaxKubernetesVersion at some point as well?

Reply:
CIS benchmark doesn't have any limitations on max, hence probably not.